service/controller/orgcluster/resource/orgclusterresources/config.go

```diff
@@ -0,0 +1,62 @@
+package orgclusterresources
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	pkgkey "github.com/giantswarm/rbac-operator/pkg/key"
+)
+
+type rolePair struct {
+	roleName        string
+	roleBindingName string
+	policyRules     []rbacv1.PolicyRule
+}
+
+// List of roles and roleBinding pairs that should be ensured as well as the granted permissions
+func referencedClusterRoles() []rolePair {
```

Comment:
This contains which roles need to be managed in the cluster namespace. It includes the desired name for role and rolebinding as well as the granted permissions. I think defining it in one place makes it easier to make changes later.